statescript/executor.go

```diff
@@ -88,12 +88,23 @@ func (l Launcher) CheckRootfsScriptsVersion() error {
 		return nil
 	}
 
-	ver, err := readVersion(filepath.Join(l.RootfsScriptsPath, "version"))
+	versionFilePath := filepath.Join(l.RootfsScriptsPath, "version")
+	f, err := os.Open(versionFilePath)
 	if err != nil && os.IsNotExist(err) {
-		// no version
-		return errors.New("statescript: missing rootfs scripts version file")
+		errmsg := `statescript: The statescript version file is missing. This file needs to be
+present and contain a single number representing which version of the statescript support the client is using
+in order to successfully run statescripts on the client`
```

Comment:
Will these turn into real newlines? I think they will end up looking quite ugly in the log (literal `\n`). Maybe break with normal spaces, and in the code, with `+` concatenated strings instead?